shared/constants/multichain/networks.ts

```diff
@@ -34,31 +39,45 @@ export type MultichainProviderConfig = ProviderConfigWithImageUrl & {
 export type MultichainNetworkIds = `${MultichainNetworks}`;
 
 export enum MultichainNetworks {
```

Comment:
Is this map really needed? All its doing is remapping the scopes values. Could we not just use the scope values as keys? This is what we have been doing on mobile.

Reply:
I agree, will check with @ccharly since he added it initially

Reply:
ok, lets keep it for now